src/mattack_actors.h

```diff
@@ -0,0 +1,32 @@
+#ifndef MATTACK_ACTORS_H
+#define MATTACK_ACTORS_H
+
+#include "json.h"
+#include "monattack.h"
```

Comment:
json.h doesn't need to be included here, forward declaring JsonObject should be enough (it's needed in the cpp file). Similar: monattack.h doesn't seem to be required either.